azure/Guidelines.md

```diff
@@ -88,80 +86,53 @@ All Azure APIs **MUST** use explicit versioning. The Microsoft REST API guidelin
-
-At a high level, changes to the contract of an API constitute a breaking change. Changes that impact backwards compatibility of an API is also considered a breaking change. Anything that would violate the Principle of Least Astonishment is considered a breaking change in Azure. Below are some concrete examples of what constitutes a breaking change. In the below breaking change scenarios, the API version must be changed.
-
-#### Existing property is removed
-
-If a property called `foo` that was present in v1 of the API needs to be removed, it must be done in a newer API version.
-
-#### New property added to response
-
-If a new property/field is added to the response of an API, the GET-PUT pipeline will be broken. Consider the case where a customer updates the value of a new property "A" from the Azure portal. Another customer does a GET of this resource using the SDK. The SDK will ignore the property since it does not understand it. From the SDK, the customer does a PUT using the model that was returned from the GET. This will overwrite the change made by the first customer from the portal.
-
-#### New required property added to request
+### API Changes that require a version change
 
-If a new property is made required in the request body, clients will have no way to set this and the request will fail.
+Azure services **MUST** update the version number of their API whenever there is a change to the API, no matter how small.  Customers will "lock the API version" so that their code does not fail when the service introduces new features.  They rely on the fact that an API version is a contract with the services that will never change.
 
-#### Property name has changed
+A breaking change is any change in the API that may cause client or service code making the API call to fail. Obvious examples of such a change are the removal of an endpoint, adding or removing a required field or changing the format of the body (from XML to JSON for example). Even though we recommend clients ignore new fields, there are many libraries and clients that fail when new fields are introduced.
 
-Note that this is implied by the requirement that adding/and removing properties are breaking changes, but in some ways worse, since it leads to the possibility of reusing a property name. Even with an API version change, this change is discouraged because it creates documentation and cognitive challenges.
+There are three reasons why a service may issue a breaking change:
```

Comment:
Are these meant to be reasons why a change could be made without changing the api-version, or are api-version changes required, even in these cases?